smrf/data/hrrr_grib.py

```diff
@@ -149,6 +160,34 @@ def parse_data(self, metadata, data):
             solar, self.metadata,
             self.topo.basin_lat, self.topo.basin_long)
 
+    def calculate_wind(self, data):
+        """
+        Calculate the wind speed and wind direction.
+
+        Args:
+            data: Loaded data from weather_forecast_retrival
+        """
+        dataframe_options = dict(
+            index=data['air_temp'].index,
+            columns=data['air_temp'].columns
+        )
+        wind_speed = np.zeros_like(data['air_temp'].values)
+        wind_direction = np.zeros_like(data['air_temp'].values)
```

Comment:
We've had a problem before of initializing with 0 values. 0 has meaning in SMRF and if something were to happen and not overwrite the dataframe, SMRF would continue on. The remedy was to initialize with NaN as that will cause SMRF to crash.